modules/series-service-impl/src/main/java/org/opencastproject/series/impl/SeriesServiceImpl.java

```diff
@@ -256,19 +189,22 @@ public DublinCoreCatalog updateSeries(DublinCoreCatalog dc) throws SeriesExcepti
         }
 
         logger.debug("Updating series {}", id);
-        index.updateIndex(dublinCore);
+        // ??? Is this even necessary? This function is about updating DC after all???
```

Comment:
The index update that is being removed is the Series service's internal Solr service. The Elasticsearch based index still exists and is still being updated.

My comment here is only about the ACL update, which doesn't make sense here since the ACL is most likely still the old one and it  is updated if you update the ACL in the series service.